commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/queries/ShippingMethodsByCartGetImpl.java

```diff
@@ -1,24 +1,31 @@
 package io.sphere.sdk.shippingmethods.queries;
 
 import com.fasterxml.jackson.core.type.TypeReference;
+import io.sphere.sdk.http.NameValuePair;
 import io.sphere.sdk.queries.MetaModelGetDslBuilder;
 import io.sphere.sdk.queries.MetaModelGetDslImpl;
 import io.sphere.sdk.shippingmethods.ShippingMethod;
 import io.sphere.sdk.shippingmethods.expansion.ShippingMethodExpansionModel;
 
+import java.util.Collections;
 import java.util.List;
 
-final class ShippingMethodsByCartGetImpl extends MetaModelGetDslImpl<List<ShippingMethod>, ShippingMethod, ShippingMethodsByCartGetImpl, ShippingMethodExpansionModel<ShippingMethod>> implements ShippingMethodsByCartGet {
+final class ShippingMethodsByCartGetImpl extends MetaModelGetDslImpl<List<ShippingMethod>, ShippingMethod, ShippingMethodsByCartGet, ShippingMethodExpansionModel<ShippingMethod>> implements ShippingMethodsByCartGet {
```

Comment:
Uh that looked wrong before 😅 or was it actually necessary with the previous implementation?

Reply:
Good catch, I didn't notice that I changed this. But I agree that it looed wrong. And it doesn't seem to make any difference if you use `ShippingMethodsByCartGet` or `ShippingMethodsByCartGetImpl`. And this type parameter `<T>` is only used for expansion paths and I think that's why the "before" version worked.